<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NSUCircle</class>
 <widget class="QDialog" name="NSUCircle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NSUCircle</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>658</width>
         <height>68</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>308</width>
           <height>168</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>321</width>
           <height>171</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>321</width>
           <height>171</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="title">
          <string>로그인</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>12</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="idLabel">
                    <property name="styleSheet">
                     <string notr="true">font: 75 14pt &quot;맑은 고딕&quot;;</string>
                    </property>
                    <property name="text">
                     <string>아이디</string>
                    </property>
                    <property name="buddy">
                     <cstring>idLineEdit</cstring>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLabel" name="passwordLabel">
                  <property name="styleSheet">
                   <string notr="true">font: 75 14pt &quot;맑은 고딕&quot;;</string>
                  </property>
                  <property name="text">
                   <string>비밀번호</string>
                  </property>
                  <property name="buddy">
                   <cstring>passwordLineEdit</cstring>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLineEdit" name="idLineEdit">
                  <property name="styleSheet">
                   <string notr="true">font: 12pt &quot;나눔바른고딕&quot;;
background-color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="clearButtonEnabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="passwordLineEdit">
                  <property name="styleSheet">
                   <string notr="true">font: 12pt &quot;나눔바른고딕&quot;;
background-color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QPushButton" name="findIdButton">
                  <property name="minimumSize">
                   <size>
                    <width>101</width>
                    <height>31</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>101</width>
                    <height>31</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
                  </property>
                  <property name="text">
                   <string>아이디 찾기</string>
                  </property>
                  <property name="autoDefault">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="findPasswordButton">
                  <property name="minimumSize">
                   <size>
                    <width>101</width>
                    <height>31</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>101</width>
                    <height>31</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 9pt &quot;맑은 고딕&quot;;</string>
                  </property>
                  <property name="text">
                   <string>비밀번호 찾기</string>
                  </property>
                  <property name="autoDefault">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>58</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>18</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="okButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>111</width>
                    <height>41</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>111</width>
                    <height>41</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
                  </property>
                  <property name="text">
                   <string>확  인</string>
                  </property>
                  <property name="autoDefault">
                   <bool>true</bool>
                  </property>
                  <property name="default">
                   <bool>true</bool>
                  </property>
                  <property name="flat">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>658</width>
         <height>58</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="joinButton">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>121</width>
           <height>51</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="text">
          <string>회원가입</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>88</width>
           <height>48</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="translateComboBox">
             <property name="minimumSize">
              <size>
               <width>101</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>101</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
             </property>
             <item>
              <property name="text">
               <string>한국어</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>English</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="translateButton">
             <property name="minimumSize">
              <size>
               <width>81</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>81</width>
               <height>31</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;맑은 고딕&quot;;</string>
             </property>
             <property name="text">
              <string>번  역</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>88</width>
           <height>48</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>131</width>
           <height>51</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;맑은 고딕&quot;;</string>
         </property>
         <property name="text">
          <string>종  료</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
